### PR TITLE
Gallery section is implemented.

### PR DESCRIPTION
**Pull request by iamsoconfident**
Gallery section is implemented and commented fixes are corrected. These fixes are: 
**1)** Moving "**fonts**" folder to "**public**" folder.
**2)** Fonts and variables (background color etc.) are imported directly from "**root.scss**" file to not repeat same code each file. 
**3)** **BEM** (Block, Element and Modifier) naming convention is implemented. 
**4)** As naming convention implemented, **.scss** file's class names are also adopted in **.jsx** file. 
**5)** **About.jsx** and **Gallery.jsx** files are renamed to **index.jsx**.

### CODE REVIEW

File: src/components/organisms/About/About.scss [hunk 1]
```diff
@@ -1,54 +1,48 @@
...
-}
-
-@font-face {
-  font-family: "Axis";
-  src: url("../fonts/Axis.woff2") format("truetype");
-}
+@import "../../../root.scss";
 
-$subtitle_color: #9c674c;
+$montserrat: "Montserrat";
```
Comment (Jalal999) [inline src/components/organisms/About/About.scss:3]: you can take these to root as well

File: src/components/organisms/Gallery/Gallery.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  gap: 8px;
+  width: 100%;
+}
+
+.gallery__text {
+  margin: center;
+  font-family: $montserrat;
+  line-height: 20px;
+}
+.gallery__text--subtitle {
```
Comment (Jalal999) [inline src/components/organisms/Gallery/Gallery.scss:27]: still don't see nesting of the properties

File: src/components/organisms/Gallery/index.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import "./Gallery.scss";
+
+const Gallery = (component) => {
```
Comment (Jalal999) [inline src/components/organisms/Gallery/index.jsx:3]: why you pass component instead of props

Comment (iamsoconfident) [reply src/components/organisms/Gallery/index.jsx:3]: Yesterday commented about this issue, in this case, does "props.props" would be more suitable convention ?

File: src/components/organisms/About/About.scss (hunk 1 above)
Comment (iamsoconfident) [reply src/components/organisms/About/About.scss]: already fixed.

**Review by Jalal999 — APPROVED** [review]

**Pull request merged**